src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.ts

```diff
@@ -88,6 +91,64 @@ export class RgwBucketListComponent extends ListWithDetails {
     this.timeConditionReached();
   }
 
+  ngOnInit() {
+    this.columns = [
+      {
+        name: this.i18n('Name'),
+        prop: 'bid',
+        flexGrow: 1
+      },
+      {
+        name: this.i18n('Owner'),
+        prop: 'owner',
+        flexGrow: 2
+      },
+      {
+        name: this.i18n('Size Usage'),
+        prop: 'size_usage',
+        cellTemplate: this.bucketSizeTpl,
+        flexGrow: 1
+      },
+      {
+        name: this.i18n('Current Size'),
+        prop: 'bucket_size',
+        flexGrow: 1,
+        pipe: this.dimlessBinaryPipe
+      },
+      {
+        name: this.i18n('No. of Objects'),
+        prop: 'num_objects',
+        flexGrow: 1
```

Comment:
You should use the `dimlessPipe` here. Not to mistake with the `dimlessBinaryPipe`. The `dimlessPipe` will convert large numbers in better readable ones (1000000 => 1M).